waku.nimble

```diff
@@ -105,8 +105,14 @@ task chat2bridge, "Build chat2bridge":
 ### C Bindings
 task libwakuStatic, "Build the cbindings waku node library":
   let name = "libwaku"
-  buildLibrary name, "library/", "-d:chronicles_log_level=ERROR", "static"
+  buildLibrary name,
+    "library/",
+    """-d:chronicles_line_numbers -d:chronicles_runtime_filtering=on -d:chronicles_sinks="textlines,json" -d:chronicles_default_output_device=Dynamic -d:chronicles_disabled_topics="eth,dnsdisc.client" """,
```

Comment:
Just wonder - although I see it is the default in nim.cfg also - why exactly these and only these topics are disabled??? I have no issue with it, just seems such a leftover...